packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -797,8 +796,9 @@ mixin WidgetInspectorService {
   void registerServiceExtension({
     required String name,
     required ServiceExtensionCallback callback,
+    required RegisterServiceExtensionCallback registration,
```

Comment:
the name `registration` is a little confusing to me as the tense seems odd. I don't have a great name idea but, I would err on the side of making it clear but a bit verbose. For example:
registerServiceExtensionCallback 
alternatives:
* `onRegisterServiceExtension`
* `serviceExtensionCallback`
* `registerExtension`
* `extensionRegistrar`